File: RUST/random-rust-stuff/2.rs
use std::io;

fn main() {
    println!("Enter the first number:");
    let mut first_num = String::new();  // declare a mutable string to store user input
    io::stdin()
        .read_line(&mut first_num)
        .expect("Failed to read line");  // read user input and store it in `first_num`

    let first_num: i32 = first_num.trim().parse().expect("Please enter a valid integer"); // convert user input to integer and reassign it to `first_num`


    println!("Enter the second number:");
    let mut second_num = String::new();
    io::stdin()
        .read_line(&mut second_num)
        .expect("Failed to read line");

    let second_num: i32 = second_num.trim().parse().expect("Please enter a valid integer");



    let sum = first_num + second_num;
    println!("The sum of {} and {} is {}", first_num, second_num, sum);
}

File: C/uni-stuff/semester-2/Programming-in-C/12-3-24/2.c
// print the sum of n numbers

#include <stdio.h>

int main() {
  int n, sum = 0;
  printf("Enter the value of n: ");
  scanf("%d", &n);
  for (int i = 1; i <= n; i++) {
    sum += i;
  }
  printf("The sum of the first %d numbers is %d\n", n, sum);

  return 0;
}